src/client/common/application/notebook.ts

```diff
@@ -32,6 +33,11 @@ export class VSCodeNotebook implements IVSCodeNotebook {
                   kernel: NotebookKernel | undefined;
               }>().event;
     }
+    public get onDidChangeNotebookEditorSelection(): Event<NotebookEditorSelectionChangeEvent> {
+        return this.canUseNotebookApi
+            ? this.window.onDidChangeNotebookEditorSelection
+            : new EventEmitter<NotebookEditorSelectionChangeEvent>().event;
```

Comment:
Do we need to make sure we add these new EventEmitters to the disposable registry?

Reply:
Fixed.